app/src/components/Header/Header.styles.js

```diff
@@ -1,57 +1,203 @@
 import styled from "styled-components";
 import { NavLink } from "react-router-dom";
+import burgerMenu from "../../images/menu-burger.png";
 
 export const StyledHeader = styled.div`
-  margin: 0;
-  padding: 0;
-  padding-left: 60px;
-  width: 1440px;
-  height: 120px;
-  padding-top: 15px;
-  display: flex;
-  align-items: center;
-  justify-content: space-around;
+  @media screen and (min-width: 350px) {
+    width: 100%;
+    padding: 20px 0;
+    position: relative;
+  }
+
+  @media screen and (min-width: 768px) {
+    padding-top: 10px;
+    height: 120px;
+    margin: 0 auto;
+    width: 80%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+  }
+
+  @media screen and (min-width: 1440px) {
+    margin: 0 auto;
+    width: 80%;
+    height: 120px;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    max-width: 80rem;
+  }
 `;
 export const StyledNavigation = styled.div`
-  display: flex;
-  width: 44.44rem;
-  align-items: center;
-  justify-content: flex-start;
+  @media (min-width: 350px) {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    width: 100%;
+    margin: 0px;
+    padding: 0px;
+    .dropdown_menu {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      position: absolute;
+      padding: 30px 20px;
+      padding-top: 35px;
+      top: 100px;
+      left: 20px;
+      width: 18.5rem;
+      height: 20.5rem;
+      background-color: ${(props) => props.theme.colors.darkViolet};
+      border-radius: 10px;
+    }
+  }
+  @media screen and (min-width: 768px) {
+    display: flex;
+    width: 60%;
+    align-items: center;
+    justify-content: flex-start;
+  }
+
+  @media screen and (min-width: 1440px) {
+    display: flex;
+    width: 60%;
+    align-items: center;
+    justify-content: flex-start;
+  }
 `;
 export const StyledLogoNavLink = styled(NavLink)`
-  font-size: 28px;
-  color: ${(props) => props.theme.colors.black};
-  font-weight: 700;
-  text-decoration: none;
+  @media screen and (min-width: 350px) {
+    color: ${(props) => props.theme.colors.black};
+    font-weight: 700;
+    font-size: 2rem;
+    text-decoration: none;
+  }
+  @media screen and (min-width: 1440px) {
+    font-size: 28px;
+  }
 `;
 export const StyledNavLink = styled(NavLink)`
-  margin-left: 30px;
-  font-size: 14px;
+  @media (min-width: 350px) {
+    display: none;
+  }
+  @media screen and (min-width: 768px) {
+    display: flex;
+    margin-left: 30px;
+    font-size: 14px;
+    color: ${(props) => props.theme.colors.gray};
+    font-weight: 700;
+    text-decoration: none;
+  }
+  @media screen and (min-width: 1440px) {
+    margin-left: 30px;
+    font-size: 14px;
+    color: ${(props) => props.theme.colors.gray};
+    font-weight: 700;
```

Comment:
if you specify font weight in min-width: 768px than it's not needed in min-width: 1440, so this can be removed

Reply:
fixed.